### PR TITLE
{App Config} Fix #22529: Remove export file in test

### LINKED ISSUE #22529: `test_azconfig_import_export_naming_conventions` randomly fails
**Describe the bug**

`test_azconfig_import_export_naming_conventions` randomly fails in "Test ubuntu jammy Package".

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1587306&view=logs&j=5e86a2db-adda-5de7-5267-91d214f7a907&t=1b2a6e63-665b-5eba-3cb8-d397d5f17f4c&l=3123

```
2022-05-19T10:00:33.7485436Z =================================== FAILURES ===================================
2022-05-19T10:00:33.7486247Z _ AppConfigImportExportNamingConventionScenarioTest.test_azconfig_import_export_naming_conventions _
2022-05-19T10:00:33.7487748Z [gw0] linux -- Python 3.10.4 /opt/az/bin/python3
2022-05-19T10:00:33.7488700Z self = <command_modules.appconfig.tests.latest.test_appconfig_commands.AppConfigImportExportNamingConventionScenarioTest testMethod=test_azconfig_import_export_naming_conventions>
2022-05-19T10:00:33.7489965Z resource_group = 'clitest.rg000001', location = 'eastus'
2022-05-19T10:00:33.7490340Z 
2022-05-19T10:00:33.7490803Z     @AllowLargeResponse()
2022-05-19T10:00:33.7491553Z     @ResourceGroupPreparer(parameter_name_for_location='location')
2022-05-19T10:00:33.7492307Z     def test_azconfig_import_export_naming_conventions(self, resource_group, location):
2022-05-19T10:00:33.7493323Z         config_store_name = self.create_random_name(prefix='NamingConventionTest', length=24)
2022-05-19T10:00:33.7494188Z     
2022-05-19T10:00:33.7494913Z         location = 'eastus'
2022-05-19T10:00:33.7495617Z         sku = 'standard'
2022-05-19T10:00:33.7496134Z         self.kwargs.update({
2022-05-19T10:00:33.7496869Z             'config_store_name': config_store_name,
2022-05-19T10:00:33.7497546Z             'rg_loc': location,
2022-05-19T10:00:33.7498440Z             'rg': resource_group,
2022-05-19T10:00:33.7499308Z             'sku': sku
2022-05-19T10:00:33.7499921Z         })
2022-05-19T10:00:33.7500445Z         _create_config_store(self, self.kwargs)
2022-05-19T10:00:33.7500944Z     
2022-05-19T10:00:33.7501736Z         import_hyphen_path = os.path.join(TEST_DIR, 'import_features_hyphen.json')
2022-05-19T10:00:33.7502790Z         exported_file_path = os.path.join(TEST_DIR, 'export_features.json')
2022-05-19T10:00:33.7503784Z         export_underscore_path = os.path.join(TEST_DIR, 'export_features_underscore.json')
2022-05-19T10:00:33.7504864Z         import_multiple_feature_sections_path = os.path.join(TEST_DIR, 'import_multiple_feature_sections.json')
2022-05-19T10:00:33.7506008Z         import_wrong_enabledfor_format_path = os.path.join(TEST_DIR, 'import_wrong_enabledfor_format.json')
2022-05-19T10:00:33.7506619Z     
2022-05-19T10:00:33.7507133Z         self.kwargs.update({
2022-05-19T10:00:33.7507819Z             'import_source': 'file',
2022-05-19T10:00:33.7508549Z             'imported_format': 'json',
2022-05-19T10:00:33.7509242Z             'label': 'NamingConventionTest',
2022-05-19T10:00:33.7510020Z             'naming_convention': 'underscore',
2022-05-19T10:00:33.7510806Z             'imported_file_path': import_hyphen_path,
2022-05-19T10:00:33.7511550Z             'exported_file_path': exported_file_path
2022-05-19T10:00:33.7512090Z         })
2022-05-19T10:00:33.7512531Z         self.cmd(
2022-05-19T10:00:33.7513655Z             'appconfig kv import -n {config_store_name} -s {import_source} --path "{imported_file_path}" --format {imported_format} --label {label} -y')
2022-05-19T10:00:33.7514499Z         self.cmd(
2022-05-19T10:00:33.7515744Z             'appconfig kv export -n {config_store_name} -d {import_source} --path "{exported_file_path}" --format {imported_format} --label {label} --naming-convention {naming_convention} -y')
2022-05-19T10:00:33.7516652Z         with open(export_underscore_path) as json_file:
2022-05-19T10:00:33.7517282Z             export_underscore_path = json.load(json_file)
2022-05-19T10:00:33.7517826Z         with open(exported_file_path) as json_file:
2022-05-19T10:00:33.7518411Z             exported_kvs = json.load(json_file)
2022-05-19T10:00:33.7518994Z >       assert export_underscore_path == exported_kvs
2022-05-19T10:00:33.7519924Z E       AssertionError: assert {'Color': 'Re...:00:00Z'}}]}}} == {'Color': 'Re...', 'ship3']}}}
2022-05-19T10:00:33.7520843Z E         Omitting 2 identical items, use -vv to show
2022-05-19T10:00:33.7521434Z E         Left contains 1 more item:
2022-05-19T10:00:33.7522164Z E         {'feature_management': {'Beta': False,
2022-05-19T10:00:33.7522987Z E                                 'Percentage': True,
2022-05-19T10:00:33.7523776Z E                                 'Timestamp': {'enabled_for': [{'Name': 'Local Tests',
2022-05-19T10:00:33.7524757Z E                                                                'Parameters': {'EndTime': '2019-09-01T00:00:00Z',
2022-05-19T10:00:33.7525761Z E                                                                               'StartTime': '2019-01-01T00:00:00Z'}},...
2022-05-19T10:00:33.7526361Z E         
2022-05-19T10:00:33.7527046Z E         ...Full output truncated (44 lines hidden), use '-vv' to show
2022-05-19T10:00:33.7527491Z 
2022-05-19T10:00:33.7528456Z /opt/az/lib/python3.10/site-packages/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py:1100: AssertionError
2022-05-19T10:00:33.7529487Z ----------------------------- Captured stderr call -----------------------------
2022-05-19T10:00:33.7530609Z WARNING: Argument '--retention-days' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
2022-05-19T10:00:33.7531981Z ----------- generated xml file: /azure_cli_test_result/appconfig.xml -----------
2022-05-19T10:00:33.7532789Z =========================== short test summary info ============================
2022-05-19T10:00:33.7533640Z FAILED tests/latest/test_appconfig_commands.py::AppConfigImportExportNamingConventionScenarioTest::test_azconfig_import_export_naming_conventions
```

### PR DESCRIPTION
**Pull request by bebound**
**Description**
Fix #22529. `test_azconfig_import_export_naming_conventions` and `test_azconfig_import_export` both use `export_features.json` as export file, so the test randomly fails when run in parallel. 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "Color": "Red",
   "Region": "West US",
-  "feature_management": {
+  "FeatureManagement": {
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json:4]: What is the correct name?

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_naming.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_naming.json:1]: If this file is gerated by https://github.com/Azure/azure-cli/blob/2d84719fe003c7fae9a58393abe7be77dbd6c8d5/src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py#L1095

shouldn't the test case delete this file when test finishes?

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json (hunk 1 above)
Comment (bebound) [reply src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json:4]: As two test use the same output, these two names are both valid.

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_naming.json (hunk 2 above)
Comment (bebound) [reply src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_naming.json:1]: Good advice. I'll fix this.

**Comment by yonzhan** [comment]
App Config

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json (hunk 1 above)
Comment (avanigupta) [reply src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json:4]: We can use any name, as long as they are different. This looks good to me. Thanks for fixing it!

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_naming.json (hunk 2 above)
Comment (bebound) [reply src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_naming.json]: Yes. Removing files does not affect test.

**Review by jiasli — APPROVED** [review]

**Comment by jiasli** [comment]
Nice work!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**